### ISSUE TITLE
[ui5-list]: a11y-JAWS announce group end before group list item end in list

### ISSUE THREAD

**Issue by pankajpandey01** [SAP/ui5-webcomponents#7561]
### Bug Description

[ui5-list]: a11y-JAWS announce group end before group list item end in list
Follow the "steps to reproduce". 
Actual behavior: JAWS announce Group end when focus go to first list item checkbox

### Affected Component

ui5-list

### Expected Behaviour

Group end should be announced by JAWS when group list item end.

### Isolated Example

https://sap.github.io/ui5-webcomponents/playground/?path=/story/main-list--group-headers

### Steps to Reproduce

1. Start JAWS 2023 (VPC Mode)
2. Go to https://sap.github.io/ui5-webcomponents/playground/?path=/story/main-list--group-headers
3. Select group header "Front End Developers"
4. Press down arrow key to select list item checkbox.
..

### Priority

Medium

### UI5 Web Components Version

1.17.0

### Browser

Chrome

### Operating System

Mac OS

### Organization

SuccessFactors

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by dimovpetar** [comment]
Hello @SAP/ui5-webcomponents-topic-p,

In VPC mode, you will hear: 
1: "Group Start Front End Developers" 
2: "Front End Developers 
3: "Group End Front End Developers", although the group has not ended

Maybe this is related to the fact that the group header is an "ul" element?

Best regards,
Petar

**Comment by dobrinyonkov** [comment]
Hello, the strucutre of the groups in the list has changed with the recent 2.0 release. Currently there is a new `ui5-li-group` component that wraps the list items in an element that has the role group. Keep in mind that there is an open issue about wrong nesting of the group role opened here: https://github.com/SAP/ui5-webcomponents/issues/9242
